manifest.json

```diff
@@ -247,6 +247,17 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/ubuntu/16.04/cross-arm64",
```

Comment:
I think the best long term solution and to follow the existing convention, it would be best to have an arm64 subfolder in the cross directory instead of using a cross-arm64 directory.

Reply:
I was thinking given #16 that the cross directory would eventually go away and become cross-arm, cross-x86, and cross-arm64 respectively.

I can revise if you prefer.

Reply:
Agreed but I was thinking the cross directory would contain the 3 arch sub directories.